Jenkinsfile

```diff
@@ -85,6 +85,7 @@ podTemplate(name: 'fedora-atomic-inline', label: 'fedora-atomic-inline', cloud:
                             echo ${CI_MESSAGE} | ${WORKSPACE}/parse_fedmsg.py > fedmsg_fields.txt
                             sed -i '/^\\\\s*$/d' ${WORKSPACE}/fedmsg_fields.txt
                             sed -i '/`/g' ${WORKSPACE}/fedmsg_fields.txt
+                            sed -i '/^fed/!d\' ${WORKSPACE}/fedmsg_fields.txt
```

Comment:
Is the intent to delete all fields which don't start with fed?  Also - why is the closing quote escaped?  I don't think that will work,